nailgun/entity_mixins.py

```diff
@@ -424,7 +438,26 @@ def delete(self, synchronous=True):
 
 
 class EntityReadMixin(object):
-    """A mixin that provides the ability to read an entity."""
+    """This mixin provides the ability to read an entity.
+
+    The methods provided by this class work together. The call tree looks like
+    this::
+
+        read → read_json → read_raw
+
+    In short, here is what the methods do:
+
+    :meth:`read_raw`
+        Make an HTTP GET request to the server.
+    :meth:`read_json`
+        Check the server's response for errors and decode the response.
+    :meth:`read`
+        Create a :class:`nailgun.entity_mixins.Entity` object representing the
+        created entity and populate its fields.
```

Comment:
Do we want to say something like '...populate its fields with the data returned from the server.'?